### ISSUE TITLE
gamepad and on screen controls not working

### ISSUE THREAD

**Issue by lixryan** [moonlight-stream/moonlight-android#178]
First of all, english is not my first language so I'm sorry if I would confuse you guys/girls here with my choice of words.

So I had this issue when i plug my dualschock 4 gamepad doesn't work when launching a game for streaming but it do work for basic android function like selecting apps and launching it. So i decided to give the on screen controls some try to see if it was my dualshock 4 is the one with problems. But it doesn't work as well when i tap into the any of the on screen controls to choose or select a menu from the game i'm streaming. So i decided to clear moonlight data and re-install but doesn't yield any success. And I'm totally at lost as to what to do

I'm using :
moonlight for rooted devices version 4.5.5
Geforce Experience version 2.102.4

**Comment by Gershy13** [comment]
It works for me, its just that there is a lot of lag...

**Comment by Gershy13** [comment]
the virtual controller i mean...

**Comment by Brenttime** [comment]
I am having the same issue

**Comment by cgutman** [comment]
What game is is happening with? These issues are almost always game specific.

**Comment by Brenttime** [comment]
Non game related for me. Games i have tried include: MGS5, Fallout 4, steam, witcher 3, Rainbow six siege, skyrim, and more. I did some investigative work by trying a uplay game, a few steam games, and a gog game on purpose, but obviously to no better results. I have used a bluetooth moga, set to hid mode, hero power, a xbox 360 usb, via otg cable, and an xbox one wired, via otg cable. I tried turning on and off settings, but still no luck. I even went to see if changing the input file in program files, using steps provided, would help; sadly it did not. What i have concluded is, other than touch screen input for the mouse, no input from the stream device is being sent. The device is a samsung galaxy s6 running latest version of Android and the latest version of moonlight. The odd thing is that the bluetooth control can control the app and select the game i would like to plau, but fails, when the game launches, to control the game. Thanks for any tips anyone can provide because im at a loss.
